.ci/lib.sh

```diff
@@ -37,6 +37,12 @@ else
 	export GOPATH="${GOPATH:-$HOME/go}"
 fi
 
+if [ "$(uname -m)" == "s390x" ] && grep -Eq "\<(fedora|suse)\>" /etc/os-release 2> /dev/null; then
+	# see https://github.com/kata-containers/osbuilder/issues/217
+	export CC=gcc
```

Comment:
Not for now, but shouldn't we also expand the test to support RHEL?  I get that CentOS is out of question as CentOS doesn't provide a 390x build.

Reply:
@fidencio RHEL puts `ID_LIKE="fedora"` in its `/etc/os-release`, this test was written with that in mind.

Reply:
Perfect then, thanks @Jakob-Naucke!